### PR TITLE
V280 Fix

### PR DESCRIPTION
**Pull request by stevejameskent**
Fix for isValidMetroArea

### CODE REVIEW

File: test/testdata/array-errors.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		"lineNumber": "1",
 		"properties": {
 			"recordID": "1",
-			"filler": " "
+			"filler": ""
```
Comment (poorgeek) [inline test/testdata/array-errors.json]: Just checking that you intended to reduce the filler to an empty string.

Comment (stevejameskent) [inline test/testdata/array-errors.json]: @poorgeek The file parser was updated recently to .trim() all fields so I updated the test data to match.

Comment (poorgeek) [inline test/testdata/array-errors.json]: Ok. just looked a little odd alongside the rest of the commits and wanted to make sure

Comment (LinuxBozo) [inline test/testdata/array-errors.json]: That would be correct based on the fact that we now `.trim()` the fields in the HMDA JSON during processing.

**Comment by optilogic1** [comment]
Hi everybody, what are you working on?

**Pull request merged**